.github/comment.yml

```diff
@@ -36,11 +62,6 @@
     label: Approved-OkToMerge
```

Comment:
openapiHub link: https://aka.ms/openapihub

Reply:
Fixed in all places.